test/byContract/IexecEscrow/IexecEscrowNative.ts

```diff
@@ -0,0 +1,244 @@
+const depositArgs = [{ value: etherDepositAmount }] as [{ value: BigNumber }];
+
+// TODO: remove this when poco is also available in Native mode
+if (CONFIG.chains.default.asset === 'Native') {
+    describe('EscrowNative', () => {
+        let proxyAddress: string;
+        let [iexecPoco, iexecPocoAsAnyone]: IexecInterfaceNative[] = [];
+        let [iexecAdmin, anyoneA, anyoneB, anyone]: SignerWithAddress[] = [];
+
+        beforeEach('Deploy', async () => {
+            proxyAddress = await loadHardhatFixtureDeployment();
+            await loadFixture(initFixture);
+        });
+
+        async function initFixture() {
+            const accounts = await getIexecAccounts();
+            ({ iexecAdmin, anyone: anyoneA, requester: anyoneB, anyone } = accounts);
+
+            iexecPoco = IexecInterfaceNative__factory.connect(proxyAddress, iexecAdmin);
+            iexecPocoAsAnyone = iexecPoco.connect(anyone);
+        }
+
+        describe('Receive and Fallback', () => {
+            it('Should call receive successfully', async () => {
+                await expect(
+                    await anyoneA.sendTransaction({
+                        to: iexecPoco.address,
+                        value: etherDepositAmount,
+                    }),
+                )
+                    .to.changeEtherBalances(
+                        [anyoneA, iexecPoco],
+                        [-etherDepositAmount, etherDepositAmount],
+                    )
+                    .to.changeTokenBalances(iexecPoco, [anyoneA], [depositAmount])
+                    .to.emit(iexecPocoAsAnyone, 'Transfer')
+                    .withArgs(AddressZero, anyoneA.address, depositAmount);
+            });
+
+            it('Should call fallback successfully', async () => {
+                const randomData = ethers.utils.hexlify(
+                    ethers.utils.toUtf8Bytes((Math.random() * 0xfffff).toString(16)),
+                );
+
+                await expect(
+                    await anyoneA.sendTransaction({
+                        to: iexecPoco.address,
+                        value: etherDepositAmount,
+                        data: randomData,
+                    }),
+                )
+                    .to.changeEtherBalances(
+                        [anyoneA, iexecPoco],
+                        [-etherDepositAmount, etherDepositAmount],
+                    )
+                    .to.changeTokenBalances(iexecPoco, [anyoneA], [depositAmount])
+                    .to.emit(iexecPocoAsAnyone, 'Transfer')
+                    .withArgs(AddressZero, anyoneA.address, depositAmount);
+            });
+        });
+
+        describe('Deposits', () => {
+            it('Should deposit native tokens', async () => {
+                expect(await iexecPocoAsAnyone.callStatic.deposit(...depositArgs)).to.be.true;
+                await expect(iexecPocoAsAnyone.deposit(...depositArgs))
```

Comment:
sender is not the same actor in all deposits tests, you can harmonize

Reply:
Done here [use account A as main account when it's possible](https://github.com/iExecBlockchainComputing/PoCo/pull/99/commits/fbf2372046936868106eacad2b168c6ad55ce80f)